src/api/sso/package.json

```diff
@@ -29,7 +32,10 @@
   },
   "devDependencies": {
     "@jcoreio/wait-for-postgres": "2.0.0",
+    "@senecacdot/eslint-config-telescope": "1.1.0",
     "env-cmd": "10.1.0",
-    "nodemon": "2.0.15"
+    "eslint": "7.32.0",
+    "nodemon": "2.0.15",
+    "supertest": "6.1.6"
```

Comment:
We can remove `supertest` here, SSO doesn't use it I believe.

Reply:
supertest is used in the `/api/sso/test/e2e/routes.test.js`